__tests__/tag.test.js

```diff
@@ -107,6 +107,16 @@ describe('Tag', () => {
     expect(spy.calledOnce).to.be.true;
   });
 
+  test('calls the tag touch handler correctly on touchStart', () => {
+    const spy = sinon.spy();
```

Comment:
`const onTagClickedStub = sinon.stub()`
better to use stub here as we just want to check if the method was triggered and not the implementation of the method.

Reply:
used stub